.github/workflows/native-wsl.yml

```diff
@@ -0,0 +1,153 @@
+        fi
+        if [ ${{ matrix.node-version }} -ge 6 -a ${{ matrix.node-version }} -lt 7 ]; then
+          npm install eslint@5 --no-save --ignore-scripts
+          npm install
+          npm install eslint-import-resolver-typescript@1.0.2 --no-save
+          npm uninstall @angular-eslint/template-parser --no-save
+          npm install @typescript-eslint/parser@3 --no-save 
+        fi
+        if [ ${{ matrix.node-version }} -ge 7 -a ${{ matrix.node-version }} -lt 8 ]; then
+          npm install eslint@6 --no-save --ignore-scripts
+          npm install
+          npm install eslint-import-resolver-typescript@1.0.2 --no-save
+          npm uninstall @angular-eslint/template-parser --no-save
+          npm install typescript-eslint-parser@20 --no-save 
+        fi
+        if [ ${{ matrix.node-version }} -eq 8 ]; then
+          npm install eslint@6 --no-save --ignore-scripts
+          npm install
+          npm uninstall @angular-eslint/template-parser --no-save
+          npm install @typescript-eslint/parser@3 --no-save 
+        fi
+        if [ ${{ matrix.node-version }} -gt 8 -a ${{ matrix.node-version }} -lt 10 ]; then
+          npm install eslint@7 --no-save --ignore-scripts
+          npm install
+          npm install @typescript-eslint/parser@3 --no-save 
+        fi
+        if [ ${{ matrix.node-version }} -ge 10 -a ${{ matrix.node-version }} -lt 12 ]; then
+          npm install
+          npm install @typescript-eslint/parser@4 --no-save
+        fi
+        if [ ${{ matrix.node-version }} -ge 12 ]; then
+          npm install
+        fi
+        npm run copy-metafiles
+        npm run pretest
+        npm run tests-only
+    - name: Install dependencies for Node <= 10
+      if: matrix.node-version <= '10' && matrix.configuration == 'native'
+      run: |
+        npm install --legacy-peer-deps
+        npm install eslint@7 --no-save
+    - name: Install dependencies for Node > 10
+      if: matrix.node-version > '10'  && matrix.configuration == 'native'
+      run: npm install
+    - name: Install the latest version of nyc
+      if: matrix.configuration == 'native'
+      run: npm install nyc@latest --no-save
+    - name: Copy metafiles for Node <= 8
+      if: matrix.node-version <= 8 && matrix.configuration == 'native'
+      env:
+        ESLINT_VERSION: 6
+        TRAVIS_NODE_VERSION: ${{ matrix.node-version }}
+      run: |
+        npm run copy-metafiles
+        bash ./tests/dep-time-travel.sh 2>&1
+    - name: Copy metafiles for Node > 8
+      if: matrix.node-version > 8 && matrix.configuration == 'native'
+      env:
+        ESLINT_VERSION: 7
+        TRAVIS_NODE_VERSION: ${{ matrix.node-version }}
+      run: |
+        npm run copy-metafiles
+        bash ./tests/dep-time-travel.sh 2>&1
+    - name: Config git to allow symlinks
+      if: matrix.configuration == 'native'
+      run: |
+        git config core.symlinks true
+        git reset --hard
```

Comment:
should this be done before the checkout action?

Reply:
I don't know what its purpose is in the first place. I copied over from the `appveyor` pipeline. I'll remove it and rerun and "see if anything breaks".